setup.py

```diff
@@ -62,8 +118,8 @@
         "pillow",
         (
             "compressed-tensors==0.9.3"
-            if version_info.build_type == "release"
-            else "compressed-tensors-nightly"
+            if BUILD_TYPE == "release"
+            else "compressed-tensors"
```

Comment:
Actually "compressed-tensors" for else statement will install the release version of compressed-tensors. Maybe we should use "compressed-tensors>0.9.3" here?

Reply:
Saying `>0.9.3` here won't be enough to ensure the nightly package is installed. The job will also need to set the `--pre` option for pip (or an equivalent for uv) to allow alpha versions to be installed (the default is to not install pre-release versions that are alpha, beta, or rc). When that is done, the installer should automatically pick the latest nightly, and the version specifier would be redundant.